apps/remote_control/test/lexical/remote_control/analyzer/aliases_test.exs

```diff
@@ -247,6 +261,56 @@ defmodule Lexical.RemoteControl.Analyzer.AliasesTest do
     end
   end
 
+  describe "alias ranges" do
+    test "for a simple alias" do
+      {aliases, doc} =
+        ~q[
+          defmodule Parent do
+            alias Foo.Bar.Baz|
+          end
+        ]
+        |> scope_aliases()
+
+      assert decorate(doc, aliases[:Baz].range) =~ "  «alias Foo.Bar.Baz»"
+    end
+
+    test "for a multiple alias on one line" do
+      {aliases, doc} =
+        ~q[
+            defmodule Parent do
+              alias Foo.Bar.{Baz, Quux}|
+            end
+        ]
+        |> scope_aliases()
+
+      assert decorate(doc, aliases[:Baz].range) =~ "  «alias Foo.Bar.{Baz, Quux}»"
+      assert decorate(doc, aliases[:Quux].range) =~ "  «alias Foo.Bar.{Baz, Quux}»"
+    end
+
+    test "for a multiple alias on multiple lines" do
+      {aliases, doc} =
+        ~q[
+            defmodule Parent do
+              alias Foo.Bar.{
+                Baz,
+                Quux,
+                Other
+            }|
+            end
+        ]
+        |> scope_aliases()
+
+      assert decorate(doc, aliases[:Baz].range) =~
+               "  «alias Foo.Bar.{\n    Baz,\n    Quux,\n    Other\n}»"
+
+      assert decorate(doc, aliases[:Quux].range) =~
+               "  «alias Foo.Bar.{\n    Baz,\n    Quux,\n    Other\n}»"
+
+      assert decorate(doc, aliases[:Other].range) =~
+               "  «alias Foo.Bar.{\n    Baz,\n    Quux,\n    Other\n}»"
```

Comment:
Putting these in a loop makes it easier to see at a glance that they're all meant to be equal.

```suggestion
      for name <- [:Baz, :Quux, :Other] do
        assert decorate(doc, aliases[name].range) =~
                 "  «alias Foo.Bar.{\n    Baz,\n    Quux,\n    Other\n}»"
      end
```